pkg/node/devnode.go

```diff
@@ -405,12 +405,21 @@ func NewDevBee(logger log.Logger, o *DevOptions) (b *DevBee, err error) {
 
 	apiService := api.New(mockKey.PublicKey, mockKey.PublicKey, overlayEthAddress, logger, mockTransaction, batchStore, api.DevMode, true, true, chainBackend, o.CORSAllowedOrigins)
 
-	apiService.Configure(signer, authenticator, tracer, api.Options{
+	chunkC := apiService.Configure(signer, authenticator, tracer, api.Options{
 		CORSAllowedOrigins: o.CORSAllowedOrigins,
 		WsPingPeriod:       60 * time.Second,
 		Restricted:         o.Restricted,
 	}, debugOpts, 1, erc20)
-
+	go func() {
+		for {
```

Comment:
This goroutine needs to be stopped on shutdown. Right now its leaking. We need to have select here to select chunk or shutdown signal.

Reply:
@aloknerurkar, where can I get the `shutdown` signal from? Is there already something available in this file?

Reply:
What you can do is use for loop over the channel. Then you can close this channel in the Shutdown function. Please try to go through the code and understand how all this works.